app/src/main/java/com/onboarding/pronosticAR/data/response/WeatherForecastResponse.kt

```diff
@@ -0,0 +1,11 @@
+package com.onboarding.pronosticAR.data.response
+
+import com.google.gson.annotations.SerializedName
+
+data class WeatherForecastResponse(
+    @SerializedName("main")
+    val mainResponse: TemperatureResponse = TemperatureResponse(),
+
+    @SerializedName("weather")
+    val weatherResponse: List<DescriptionResponse> = listOf()
```

Comment:
instead of listOf you need to asign a emptyList()

Reply:
done